index.bs

```diff
@@ -262,6 +262,8 @@ interface FileSystemHandle {
   readonly attribute FileSystemHandleKind kind;
   readonly attribute USVString name;
 
+  Promise<void> remove();
```

Comment:
I wonder if we should have a "recurse: true/false" option analogous to the one removeEntry has. For files it wouldn't matter if you set it or not, but for directories it would make the difference between deleting the directory even when not empty, or only allowing deletion of empty directories.

Reply:
You're right, it would be odd to only allow recursive removals via removeEntry. I'll prototype this.